apis/pipeline/pipeline.yaml

```diff
@@ -4496,6 +4496,58 @@ paths:
                 500:
                     $ref: '#/components/responses/InternalServerError'
 
+    '/api/v1/orgs/{orgId}/clusters/{id}/namespaces':
+        get:
+            security:
+                -
+                    bearerAuth: []
+            tags:
+                - clusters
+            summary: Lists namespaces for a cluster
+            description: Lists namespaces for a cluster
+            operationId: ListNamespaces
+            parameters:
+                -
+                    name: orgId
+                    in: path
+                    required: true
+                    description: Organization identification
+                    schema:
+                        type: integer
+                -
+                    name: id
+                    in: path
+                    description: Selected cluster identification (number)
+                    required: true
+                    schema:
+                        type: integer
+            responses:
+                '200':
+                    description: Namespaces listed
+                    content:
+                        application/json:
+                            schema:
+                                type: array
```

Comment:
Returning an array directly is generally not a good idea.

Reply:
thx, fixed.